app/controllers/api/v1/search_results_controller.rb

```diff
@@ -44,6 +46,15 @@ def upload_form_params
       def render_success(serializer:, status:)
         render json: serializer, status: status
       end
+
+      def filter_url_equals(search_result_list)
+        url = params.dig(:filter, :url_equals)
```

Comment:
It would be more secure to use strong params:

```ruby
def filter_params
   params.requires(filter).permit(:url)
end 
```

Reply:
Hmmm, that would be nice, but the `filter` parameter isn't always required.

For example, we still want to be able to call `/api/v1/search_results` 

Having that said, can we keep it as it is? 🙏

Reply:
@Tuubz Could you try something like:
```
def filter_params
  params.fetch(:filter, {}).permit(:url)
end 
```